inst/automation/last_commits.R

```diff
@@ -0,0 +1,31 @@
+library(gert)
```

Comment:
shouldn't this be a function in the package that'd be called from your script? Just so we're sure dependencies like gert are listed in DESCRIPTION?

Reply:
hmm, but if gert isn't needed in makeregistry, then doesn't seem like we should include it as a dependency

Reply:
but how does one get an overview of all dependencies? and why couldn't this be another function? genuine questions, I am not that stubborn :-)

Reply:
it was just easier to have a one of script, we could bring it into the pkg